config.m4

```diff
@@ -5,6 +5,13 @@ PHP_ARG_ENABLE([mongodb],
                                [Enable MongoDB support])])
 
 if test "$PHP_MONGODB" != "no"; then
+  dnl Enable C99 (required for libmongoc 1.24+)
+  AC_PROG_CC_C99
```

Comment:
Note: this was [deprecated in autoconf-2.70](https://www.gnu.org/software/autoconf/manual/autoconf-2.70/html_node/Obsolete-Macros.html) but is still usable. See [autoconf-2.60 docs](https://www.gnu.org/software/autoconf/manual/autoconf-2.60/html_node/C-Compiler.html) for details about how it works.